### PR TITLE
Implement 'query-account' tool

### LINKED ISSUE #19: Implement tool for querying the chain and filter out transactions affecting a specific account
**Task description**

To exercise the Data API, the tool should traverse the chain and print out which transactions in which blocks affect a specific account that is given as a parameter.

The tool should accept a range of block heights that default to the entire chain.

### PR DESCRIPTION
**Pull request by bisgardo**
## Purpose

Exercise the Data API by iteratively querying blocks and list transactions related to a given account.

## Changes

Implement the new tool.

Resolves #19.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by bisgardo** [comment]
Tested by full testnet chain traversal of a test account.

File: tools/query-account/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    #[clap(long = "url", help = "URL of Rosetta server.", default_value = "http://localhost:8080")]
+    url:     String,
+    #[clap(
+        long = "network",
+        help = "Network name. Used in network identifier.",
+        default_value = "testnet"
+    )]
+    network: String,
+    #[clap(long = "address", help = "Address of the account.")]
+    address: String,
```
Comment (abizjak) [inline tools/query-account/src/main.rs:24]: Use `AccountAddress` type instead. It implements FromStr so can be used
http://developer.concordium.software/concordium-rust-sdk/id/types/struct.AccountAddress.html#impl-FromStr

File: tools/query-account/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    client: Client,
+    base_url: &Url,
+    network_id: NetworkIdentifier,
+    from_block_height: i64,
+    to_block_height: i64,
+    address: String,
+) -> Result<()> {
+    for block_height in from_block_height..=to_block_height {
+        if block_height % 100 == 0 {
+            eprintln!("Querying block at height {}...", block_height);
```
Comment (abizjak) [inline tools/query-account/src/main.rs:67]: I'd probably use logging for this, but since this is a test tool it's fine.

**Review by abizjak — APPROVED** [review]
Looks good.

Since this is IO bound I would have probably preferred an async one with parallel queries (using futures::join_all) to make it faster to traverse the blocks. 

But this is OK as a test tool.

File: tools/query-account/src/main.rs (hunk 1 above)
Comment (bisgardo) [reply tools/query-account/src/main.rs:24]: The project doesn't depend on the SDK so doesn't have that type available.

File: tools/query-account/src/main.rs (hunk 2 above)
Comment (bisgardo) [reply tools/query-account/src/main.rs:67]: Yea I opted for simplicity because it's a test tool.

**Pull request merged**